### PR TITLE
Tf slim

### PR DESCRIPTION
**Pull request by adbeda**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/tf_slim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    # Also skip on OSX due to OSX version requirements, see
+    # https://github.com/conda-forge/tensorflow-feedstock/pull/62"
+    - tf_slim  # [win]
+  requires:
+    - tensorflow >=1.12.0
+
+about:
+  home: https://github.com/Maxar-Corp/ratpac/tree/tf_slim/TF-slim
+  license: Apache 2.0
+  license_file: LICENSE
```
Comment (scopatz) [inline recipes/tf_slim/meta.yaml]: license_family should be APache

File: recipes/tf_slim/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    directory contains code for training and evaluating several
+    widely used Convolutional Neural Network (CNN) image
+    classification models using TF-slim.
+
+
+extra:
+  recipe-maintainers:
+    - adbeda
+    - agcopenhaver
+    - tata17
```
Comment (scopatz) [inline recipes/tf_slim/meta.yaml]: @agcopenhaver @tata17 are you OK with being maintainers here?

Comment (adbeda) [reply recipes/tf_slim/meta.yaml]: @scopatz we're on the same team. i thought they could help on this in the future as well.

Comment (scopatz) [reply recipes/tf_slim/meta.yaml]: yep that makes sense, but we still need their explicit approval

Comment (Tata17) [reply recipes/tf_slim/meta.yaml]: @scopatz Good with me being a maintainer.

**Comment by agcopenhaver** [comment]
@scopatz happy to be a maintainer

File: recipes/tf_slim/meta.yaml (hunk 2 above)
Comment (agcopenhaver) [reply recipes/tf_slim/meta.yaml]: @scopatz happy to be a maintainer

**Comment by scopatz** [comment]
Thanks all!

**Pull request merged**